docs/en/academic/custom-templates.md

```diff
@@ -48,6 +48,14 @@ $body$
 
 Pandoc will replace this variable with the parsed contents of your Markdown file(s). If you leave it out, your content will not appear in the output file.
 
+## Activate your template
```

Comment:
I'm curious whether we might be able to find a better way to phrase it, since it's not so much about "activating" something as it is to simply point Zettlr to that file, isn't it …?

Reply:
From Zettlr's perspective, I agree with you. But from User's perspective, it's different – I (as a beginning user) have concocted a LaTeX template after much Googling. And then, it hasn't been clear from the docs how to get Zettlr to use it. This was my experience. The hack I found was to create the folder /Users/me/.local/share/pandoc/templates and put the template in there. Obvs, the `template:` instruction in Defaults File is better. So from User's perspective, I think it does seem like 'activating' the template.

Reply:
Okay, fair enough! I see your point, so let's keep it for now and in case someone complains we can still change that later :)